File: DKS-API/DTOs/SP202.cs
using System;
using DKS_API.Helpers;

namespace DKS_API.DTOs
{
    public class SP202 : PaginationParams
    {
        public string season { get; set; }
        public string brand{ get; set; }
        public string modelName{ get; set; }
        public string modelNo{ get; set; }
        public string article{ get; set; }
    }
}

File: DKS-SPA/src/app/core/_models/s_p202.ts
import { Pagination } from "./pagination";
import { utilityConfig } from "../utility/utility-config";

export class SP202 extends Pagination {

  season: string;
  brand:string;
  modelName: string;
  modelNo:string;
  article:string;
  /**
   *default set of searching parameters
   */
  constructor() {
    super();
    this.isPaging = false; //開分頁
    this.season = "";
    this.brand = "";
    this.modelName = "";
    this.modelNo = "";
    this.article = "";   
  }

  public setPagination(pagination: Pagination) {
    this.currentPage = pagination.currentPage;
    this.itemsPerPage = pagination.itemsPerPage;
    this.totalItems = pagination.totalItems;
    this.totalPages = pagination.totalPages;
  }
}
